.changeset/ten-crabs-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react-geo": patch
+---
+
+fix: unpin maplibre-gl-js-amplify
```

Comment:
should we reference the issue we are fixing here?

Reply:
good point